noir/noir-repo/compiler/integration-tests/web-test-runner.config.mjs

```diff
@@ -26,10 +26,9 @@ export default {
     // playwrightLauncher({ product: "webkit" }),
     // playwrightLauncher({ product: "firefox" }),
   ],
-  middleware: [async function setGzHeader(ctx, next) {
-    if (ctx.url.endsWith('.gz')) {
-      ctx.set('Content-Encoding', 'gzip');
-      ctx.res.removeHeader('Content-Length');
+  middleware: [async (ctx, next) => {
```

Comment:
@TomAFrench this should be compatible with both the `aztec-packages` linked bb.js and in noir repo with the npm one 🙏